### PR TITLE
task: remove npx and npm package publishing

### PR DESCRIPTION
**Pull request by mjhilton**
Replaces existing NPM Package/NPX publish and run workflow with one that uses Git and a local run experience instead, to remove the complexity of publishing and updating NPM package versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,24 +24,29 @@ Get started in minutes:
...
 ### Ongoing Usage
 
 In future:
 
 | Step | Run | Why |
 | --- | --- | --- |
-| 1. | `npm update beanni` | Update Beanni to the latest version |
-| 2. | `npx beanni fetch` | Grab your data |
-| 3. | `npx beanni explore` | Launch the analysis UI |
+| 2. | `npm run fetch` | Grab your data |
```
Comment (tathamoddie) [inline README.md:48]: Numbering

File: package.json [hunk 2]
```diff
@@ -1,13 +1,16 @@
 {
   "name": "beanni",
-  "version": "0.1.25",
+  "version": "0.1.24",
```
Comment (tathamoddie) [inline package.json:3]: Not that it matters much now, but why is the version rolling backwards?

File: package.json [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
   "scripts": {
+    "postinstall": "npm run build",
     "build": "tsc",
     "lint": "tslint --project tsconfig.json",
-    "verify": "npm-run-all build lint",
-    "prepack": "npm run verify"
+    "start": "node dist/app.js",
+    "init": "npm start -- init",
+    "fetch": "npm start -- fetch",
+    "explore": "npm start -- explore"
```
Comment (tathamoddie) [inline package.json:14]: I think `npm serve` might be a convention? Maybe we add that / rename accordingly later.

**Review by tathamoddie — COMMENTED** [review]
Yay!

File: package.json (hunk 3 above)
Comment (mjhilton) [reply package.json:13]: Not that I can find reference to. 
`npm start` is a thing, which is a shortcut for `npm run start`. Maybe that's what you're thinking of?

Would need to figure out whether `start` should do:
1. Fetch
2. Explore
3. Fetch && Explore

Sounds like for later.

File: package.json (hunk 2 above)
Comment (mjhilton) [reply package.json]: Don't know, I pulled before branching. Doesn't make much sense to me. Have brought it back to at least not revert.

**Pull request merged**